master/docs/manual/installation/misc.rst

```diff
@@ -115,33 +115,55 @@ Upgrade pip, setuptools and wheel
    cd c:\Python39
    python.exe -m pip install --upgrade pip setuptools wheel
 
-Install, create and activate virtualenv
-```````````````````````````````````````
+Install, create and activate virtualenv [optional]
+``````````````````````````````````````````````````
+This step is optional and may depend on your needs.
 
 .. code-block:: bat
 
-   python.exe -m pip install virtualenv==16.7.10
-   Scripts\virtualenv.exe --no-site-packages c:\bbw\sandbox
-   cd c:\bbw
+   python.exe -m pip install virtualenv>=1.7
+   virtualenv.exe c:\bbw\sandbox
    c:\bbw\sandbox\Scripts\activate
 
-
-Install pywin32 with system wide installation
-`````````````````````````````````````````````
+Install pywin32
+```````````````
 
 .. code-block:: bat
 
   python.exe -m pip install pywin32
+
+If you are in an activated virtualenv, finish up like this:
+
+.. code-block:: bat
+
```

Comment:
with modern pywin32, this is not needed anymore.

see the CI for best known method:

https://github.com/buildbot/buildbot/blob/master/appveyor.yml#L21

Reply:
Ok, then I'll drop this part